app/src/organisms/QuickTransferFlow/QuickTransferAdvancedSettings/FlowRate.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react'
+import { createPortal } from 'react-dom'
+import { useTranslation } from 'react-i18next'
+import {
+  Flex,
+  SPACING,
+  DIRECTION_COLUMN,
+  ALIGN_CENTER,
+  POSITION_FIXED,
+  COLORS,
+} from '@opentrons/components'
+import type { SupportedTip } from '@opentrons/shared-data'
+import { getTipTypeFromTipRackDefinition } from '@opentrons/shared-data'
+
+import { getTopPortalEl } from '../../../App/portal'
+import { ChildNavigation } from '../../ChildNavigation'
+import { InputField } from '../../../atoms/InputField'
+import { NumericalKeyboard } from '../../../atoms/SoftwareKeyboard'
+
+import { ACTIONS } from '../constants'
+import type {
+  QuickTransferSummaryState,
+  QuickTransferSummaryAction,
+  FlowRateKind,
+} from '../types'
+
+interface FlowRateEntryProps {
+  onBack: () => void
+  state: QuickTransferSummaryState
+  dispatch: React.Dispatch<QuickTransferSummaryAction>
+  kind: FlowRateKind
+}
+
+export function FlowRateEntry(props: FlowRateEntryProps): JSX.Element {
+  const { onBack, state, dispatch, kind } = props
+  const { i18n, t } = useTranslation(['quick_transfer', 'shared'])
+  const keyboardRef = React.useRef(null)
+
+  const [flowRate, setFlowRate] = React.useState<number | null>(
+    kind === 'aspirate'
+      ? state.aspirateFlowRate
+      : kind === 'dispense'
+      ? state.dispenseFlowRate
+      : null
+  )
+
+  // use lowVolume for volumes lower than 5ml
+  const tipType = getTipTypeFromTipRackDefinition(state.tipRack)
+  const flowRatesForSupportedTip: SupportedTip | undefined =
+    state.volume < 5
+      ? state.pipette.liquids.lowVolumeDefaults.supportedTips[tipType]
+      : state.pipette.liquids.default.supportedTips[tipType]
+  const minFlowRate = 0.1
+  const maxFlowRate =
+    flowRatesForSupportedTip?.uiMaxFlowRate !== undefined
+      ? flowRatesForSupportedTip?.uiMaxFlowRate
+      : Infinity
```

Comment:
nit: the `in` operator is a bit cleaner imo

```suggestion
  const maxFlowRate =
    flowRatesForSupportedTip != null &&  'uiMaxFlowRate' in flowRatesForSupportedTip
      ? flowRatesForSupportedTip.uiMaxFlowRate
      : Infinity
```

Reply:
or actually, maybe this is cleaner:

`const maxFlowRate = flowRatesForSupportedTip?.uiMaxFlowRate ?? Infinity`

Reply:
Should we use a safer alternative than `Infinity` in case this value doesn't exist? Or are we confident that it always will? If for some reason we get into a situation where the tiprack we've selected doesn't have this object populated for this pipette (which should never happen) I think putting an arbitrary value that we know are pipettes can handle would be safer.

Reply:
the value will not exist for GEN1 pipettes. I spoke with Andy sigler and Seth about this when i was refining PD flowRates and Andy said GEN1s don't have a max flow rate so infinity was ok to set.

Reply:
Got it, that makes sense! since quick transfer is only used for Flex, only GEN3 pipettes will be relevant here. I bet if we get into a situation where we have a pipette/tip mismatch causing the flowRatesForSupportedTip object to be null, analysis would likely fail for incompatible tip anyway.

Reply:
oh that is a good point, i forgot that only GEN3 pipettes will be used for this. in that case, we can use `0` instead? Either way.